django/forms/fields.py

```diff
@@ -290,8 +290,13 @@ def widget_attrs(self, widget):
 class FloatField(IntegerField):
     default_error_messages = {
         'invalid': _('Enter a number.'),
+        'step_size': _('Ensure number is of correct step size.'),
     }
 
+    def __init__(self, *, max_value=None, min_value=None, step_size=None, **kwargs):
+        self.step_size = step_size
+        super().__init__(max_value=max_value, min_value=min_value, **kwargs)
```

Comment:
```python
        if step_size is not None:
            self.validators.append(validators.StepValueValidator(step_size))
```

Use `django.core.validators.MinValueValidator` as a blueprint and create a `StepValueValidator`. Put the validation code from `to_python` in there.

Reply:
Have a look at the `IntegerField` how it's done there.
This has the advantage, that your `StepSizeValidator` can be reused.

And it shall be reused, because `step` can also be applied to an `IntegerField`.

Reply:
Done :heavy_check_mark: